### ISSUE TITLE
Feature request: Embed Tweets

### ISSUE THREAD

**Issue by rydmike** [invertase/docs.page#174]
## New feature request - Embedded tweets

It would be useful if there was a feature to embed tweets, the same way you can use a custom component to embed YouTube videos.

Currently I have to make them as image screen shoots of the tweet and set a link on the the image to the tweet. This works, but is not as handy or nice. Not super critical, I can work with the image + link too.  I would just prefer to embed a tweet, like below. This does does not work on GitHub either, but the code Twitter gives to embed a tweet is below:

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">FlexColorScheme V5-dev.2 is here 💙<br>With an even cooler config/code gen app <a href="https://t.co/3cgTNvoahH">https://t.co/3cgTNvoahH</a><br>Get the <a href="https://twitter.com/hashtag/Flutter?src=hash&amp;ref_src=twsrc%5Etfw">#Flutter</a> package here <a href="https://t.co/oDheabOXIv">https://t.co/oDheabOXIv</a> Official v5 release coming when doc update is done.<br>This thread is still relevant and has more info:<a href="https://t.co/dS8jPAB53q">https://t.co/dS8jPAB53q</a> <a href="https://t.co/gxdCAjuJxl">pic.twitter.com/gxdCAjuJxl</a></p>&mdash; Mike Rydstrom 💙 (@RydMike) <a href="https://twitter.com/RydMike/status/1511166732496429058?ref_src=twsrc%5Etfw">April 5, 2022</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>

And it should end up looking like the image plus link version below. Well almost, this URL shows what the above should look like:

https://publish.twitter.com/?query=https%3A%2F%2Ftwitter.com%2FRydMike%2Fstatus%2F1511166732496429058&widget=Tweet

But close enough to this version:

[<img src="https://user-images.githubusercontent.com/39990307/163257406-2db42b80-b6f1-4a51-adcd-a18d7f8937fd.png">](https://twitter.com/RydMike/status/1511166732496429058?s=20&t=MV6gzNO0p2lZhr-JgQycIg)

**Comment by cabljac** [comment]
this would be a nice feature. As you say, we could simply add a custom Twitter component

**Comment by rydmike** [comment]
Cool, works well great! 
Tested it here: https://docs.page/rydmike/docs_page_test/more_intro

Using:

```html
<Tweet id="1517005624814493696" width="550" align="right" cards="show" theme="dark"/>
```

Would of course be useful if there was a way to link the theme mode of the tweet light/dark setting to a prop that is tied to effective light/dark mode used on the site.

Also I did not see any difference when applying `align` prop, it always seemed to be centered.
